Dynatrace/dynatrace-operator
# Collect dynakube manifests

# Description

Add K8S manifests for DynaKubes to support archive.

## How can this be tested?
* Deploy operator and a DynaKube of your choice
* run the support archive subcommand
* check that the manifest for your DynaKube is part of the support archive.

## Checklist
